clang/tools/clang-offload-bundler/ClangOffloadBundler.cpp

```diff
@@ -84,13 +84,22 @@ static cl::opt<std::string> FilesType(
              "  o   - object\n"
              "  oo  - object; output file is a list of unbundled objects\n"
              "  gch - precompiled-header\n"
-             "  ast - clang AST file"),
+             "  ast - clang AST file\n"
+             "  aoco - unlinked Intel FPGA object file\n"
```

Comment:
Are we taking advantage of -type=aoc*?  All bundle/unbundles being done by the driver use type=o with the appropriate triple.

Reply:
removed.